process/headerCheck/headerSignatureVerify.go

```diff
@@ -120,6 +127,26 @@ func (hsv *HeaderSigVerifier) VerifySignature(header data.HeaderHandler) error {
 	return verifier.Verify(hash, bitmap)
 }
 
+func (hsv *HeaderSigVerifier) verifyConsensusSize(consensusPubKeys []string, header data.HeaderHandler) error {
+	consensusSize := len(consensusPubKeys)
+	bitmap := header.GetPubKeysBitmap()
+	numOfOnesInBitmap := 0
+	for index := range bitmap {
+		numOfOnesInBitmap += bits.OnesCount8(bitmap[index])
+	}
+
+	minNumRequiredSignatures := consensusSize*2/3 + 1
+	if numOfOnesInBitmap >= minNumRequiredSignatures {
+		return nil
+	}
+
+	log.Warn("not enough signatures",
```

Comment:
Might leave this as Debug, otherwise someone could "log flood" the other peers. 
Added task to create unit tests for this file.

Reply:
done